test/blackbox-tests/test-cases/enabled_if/non-ascii.t

```diff
@@ -0,0 +1,13 @@
+Afer the parsing non-ascii characters could be compared through expressions.
+Linked to the issue #9728
+
+  $ echo '(lang dune 3.0)' > dune-project
+  $ cat >dune <<"EOF"
+  > (rule
+  >  (alias default)
```

Comment:
nice trick to use the default alias here

Reply:
Thanks, but It was inspired by `/test/blackbox-tests/test-cases/enabled_if/read-macro.t`.